app/src/main/kotlin/cz/covid19cz/erouska/db/AppDatabase.kt

```diff
@@ -3,7 +3,7 @@ package cz.covid19cz.erouska.db
 import androidx.room.Database
 import androidx.room.RoomDatabase
 
-@Database(entities = [ScanDataEntity::class], version = 7, exportSchema = false)
+@Database(entities = [ScanDataEntity::class], version = 8, exportSchema = false)
```

Comment:
Tohle smaže starou databázi že jo? Tzn. nebudeme to moct vydat jako update, pokud stávající verze půjde ven.

Reply:
Ano museli bysme v tom pripade napsat migraci.